features/validations.ts

```diff
@@ -8,6 +8,14 @@ export const isValidPassword = (password: string) => {
   return passwordRegex.test(password);
 };
 
+export const isValidName = (name: string) => {
+  if (name === '') {
+    return false;
+  } else {
+    return true;
+  }
+};
```

Comment:
vc pode usar o [Double NOT](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_NOT#double_not_!!) pra fazer essa verificação

```suggestion
export const isValidName = (name: string) => {
  return !!name;
};
```